src/components/Header/UserMenu.jsx

```diff
@@ -9,7 +9,10 @@ import i18next from 'i18next';
 
 import { LogOutIcon } from '@/assets';
 
+import { CustomModal } from '../CustomModal,/CustomModal';
+
 export const UserMenu = () => {
+  const [isDeleteModalOpen, setDeleteModalOpen] = useState(false);
```

Comment:
use the chakra hook for managing open state instead

Reply:
fixed

Reply:
fixed but not pushed from what i see))